.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: 'bug'
+assignees: 'black-bunny-brigade'
+
+---
+
+**IMPORTANT: please make sure you ask yourself all intro questions and fill all sections of the template.**
+
+**Before we start...:**
+
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+
+**Version, Branch, or Commit:**
+
+Inform what version, branch, commit of Skunk you are using.
```

Comment:
it says Skunk here

Reply:
Do you think we should add something more general like "the project" instead?

Reply:
yes, that should make it easier to reuse 👍